x-pack/test_serverless/api_integration/test_suites/common/platform_security/api_keys.ts

```diff
@@ -68,7 +68,7 @@ export default function ({ getService }: FtrProviderContext) {
 
           ({ body, status } = await supertestWithoutAuth
```

Comment:
Looks like this one was missed
```suggestion
          ({ body, status } = await supertestAdminWithCookieCredentials
```

Reply:
But this scenario is testing the case without auth: we suppose to get 401.

On `main` we don't set API key header https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/api_integration/test_suites/common/platform_security/api_keys.ts#L69-L79

Should I keep it as is?

Reply:
Ah I see. I think that was maybe unintentional, looking at the comment 
> // expect a rejection because we're not using the internal header

I think the test was intended to fail for not utilizing the internal header, like with the other test cases.
But, this is on our team, not yours. I don't want to hold up your PR. I can fix this when I open a PR to adjust the access level for some of our other endopints.